site/cypress/e2e/Cypress-Gradeable/grade_inquiries.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Test cases revolving around grade inquires', () => {
+    beforeEach(() => {
+        cy.visit(['sample']);
+        cy.login();
+    });
+    const setGradeInquiriesForGradeable = (gradeableId, date = null, allowed = true) => {
+        cy.get(`#${gradeableId} .fa-pencil-alt`).click();
+
+        if (allowed) {
+            cy.get('[data-testid="yes-grade-inquiry-allowed"]').click();
+        }
+        else {
+            cy.get('[data-testid="no_grade_inquiry_allowed"]').click();
+        }
+
+        if (date) {
```

Comment:
Is there a reason for the branching if this function is only being called once?

Reply:
Actually I am taking reference from Selenium Tests, and since it uses branching, I have used it here. If it is unnecessary I will change it.